homeassistant/components/homekit/__init__.py

```diff
@@ -154,13 +154,15 @@ def __init__(self, hass, port, entity_filter, entity_config):
 
     def setup(self):
         """Setup bridge and accessory driver."""
-        from .accessories import HomeBridge, HomeDriver
+        from .accessories import HomeAccessory, HomeBridge, HomeDriver
 
         self._hass.bus.async_listen_once(
             EVENT_HOMEASSISTANT_STOP, self.stop)
 
+        HomeAccessory.hass = HomeBridge.hass = self._hass
```

Comment:
We should not do this.

All of Home Assistant is coded in a way that binds all gobal state to the `hass` object. Class variables go against this. Pass it into the constructor or set it as an instance variable. Don't set it as a class variable.